Modules/Microsoft365DSC/DSCResources/MSFT_EXOQuarantinePolicy/MSFT_EXOQuarantinePolicy.psm1

```diff
@@ -518,17 +617,37 @@ function Export-TargetResource
         $i = 1
         foreach ($QuarantinePolicy in $QuarantinePolicies)
         {
-            Write-Host "    |---[$i/$($QuarantinePolicies.length)] $($QuarantinePolicy.Identity)" -NoNewline
-
-            $Params = @{
-                Identity              = $QuarantinePolicy.Identity
-                Credential            = $Credential
-                ApplicationId         = $ApplicationId
-                TenantId              = $TenantId
-                CertificateThumbprint = $CertificateThumbprint
-                CertificatePassword   = $CertificatePassword
-                Managedidentity       = $ManagedIdentity.IsPresent
-                CertificatePath       = $CertificatePath
+            if ($QuarantinePolicy.QuarantinePolicyType -eq 'GlobalQuarantineTag')
+            {
+                Write-Host "    |---[$i/$($QuarantinePolicies.length)] $($QuarantinePolicy.Identity)" -NoNewline
+
+                $Params = @{
+                    Identity              = $QuarantinePolicy.Identity
+                    Credential            = $Credential
+                    ApplicationId         = $ApplicationId
+                    TenantId              = $TenantId
+                    CertificateThumbprint = $CertificateThumbprint
+                    CertificatePassword   = $CertificatePassword
+                    Managedidentity       = $ManagedIdentity.IsPresent
+                    CertificatePath       = $CertificatePath
+                    QuarantinePolicyType                        = $QuarantinePolicy.QuarantinePolicyType
+                }
+            }
+            else
```

Comment:
Why do we need an else statement here? What would be the issue with passing in QuarantinePolicyType for all scenarios, even if it's not equal to 'GlobalQuarantineTag'? The Get-TargetResource should then be handling it correctly no matter what.

Reply:
You are right, Nik. I removed the redundant if statement since it is not necessary there